paging/build.gradle.kts

```diff
@@ -15,7 +15,8 @@ version = Deps.mokoPagingVersion
 
 dependencies {
     commonMainImplementation(Deps.Libs.MultiPlatform.coroutines)
-    commonMainImplementation(Deps.Libs.MultiPlatform.mokoMvvm.common)
+    commonMainImplementation(Deps.Libs.MultiPlatform.mokoMvvmLiveData.common)
+    commonMainImplementation(Deps.Libs.MultiPlatform.mokoMvvmState.common)
```

Comment:
also please change it to `commonMainApi` it's transitive dependency (we use classes from this libs in our public api)